src/utils/secureAccount.ts

```diff
@@ -106,6 +106,14 @@ const makeSignature = async (token: { address: string; time: number }) => {
   return buffer.toString('base64')
 }
 
+export const makeDiscoverySignature = async (hotspotAddress: string) => {
+  const keypair = await getKeypair()
+  if (!keypair) return
+  const buffer = await keypair.sign(hotspotAddress)
```

Comment:
do we want the signature to have any other details in it? like a timestamp or an id of the discovery request? or is discovery eligibility determined elsewhere? basically just getting at this signature being valid indefinitely